packages/components/src/tools-panel/tools-panel-item/component.tsx

```diff
@@ -18,12 +18,17 @@ const ToolsPanelItem = (
 	props: WordPressComponentProps< ToolsPanelItemProps, 'div' >,
 	forwardedRef: Ref< any >
 ) => {
-	const { children, isShown, ...toolsPanelItemProps } = useToolsPanelItem(
-		props
-	);
+	const {
+		children,
+		isShown,
+		shouldRenderPlaceholder,
+		...toolsPanelItemProps
+	} = useToolsPanelItem( props );
 
 	if ( ! isShown ) {
-		return null;
+		return shouldRenderPlaceholder ? (
```

Comment:
Interesting, so the idea is that we always render something in the Fill to ensure that the order is preserved. Can it ever happen that all fills are not shown?